docs/static/security/api-keys.asciidoc

```diff
@@ -2,8 +2,145 @@
 [[ls-api-keys]]
 ==== Grant access using API keys
 
-
 Instead of using usernames and passwords, you can use API keys to grant
 access to {es} resources. You can set API keys to expire at a certain time,
 and you can explicitly invalidate them. Any user with the `manage_api_key`
 or `manage_own_api_key` cluster privilege can create API keys.
+
+Note that API keys are tied to the cluster they are created in. If you are
+sending output to different clusters, be sure to create the correct kind of API
+key. 
+
+NOTE: For security reasons, we recommend using a unique API key per {ls} instance.
+You can create as many API keys per user as necessary.
+
+
+[float]
+[[ls-create-api-key]]
+===== Create an API key 
+
+You can create API keys using either the
+{ref}/security-api-create-api-key.html[Create API key API] or the
+{kibana-ref}/api-keys.html[Kibana UI]. This section walks you through creating
+an API key using the {ref}/security-api-create-api-key.html[Create API key API].
+The privileges needed are the same for either approach.
+
+Here is an example that shows how to create an API key for publishing to {es}.
```

Comment:
let's clarify here that it's for the elasticsearch output ? otherwise users may wonder why permissions like `monitor` are needed just for "publishing".

Reply:
Can publish to ES in a way that that doesn't use the ES output?